doc/ref_model/chapters/chapter04.md

```diff
@@ -302,7 +302,7 @@ Table 4-13 Shows performance measurement capabiltities
 |--------------------|------------------|---------|-------------------------------------------|
 | e.vim.cap.006 | Virtual resources inventory per tenant | Yes/No | Capability to provide information related to allocated virtualised resources per tenant |
 | e.vim.cap.007 | Resources Monitoring | Yes/No | Capability to notify state changes of allocated resources |
-| e.vim.cap.008 | Virtual resources Performance  | Yes/No | Capability to collect and expose performance information on virtualised resources allocated |
+| e.vim.cap.008 | Virtual resources Performance  | Yes/No | Capability to collect and expose performance information on virtualised resources allocated  <br> **Note:** In a Kubernetes based infrastructure only CPU and memory resources are exposed.|
```

Comment:
@CsatariGergely I read this as we need to add process-level PMs and/or container-relevant PMs, no? Would the addition of such PMs change the proposed qualification? At a higher level, given this line item only identifies that PM data is collected and exposed to the workload context, but does not imply support for any specific/individual PM, is the qualification actually required and would it be more appropriate in a container-specific area of the RM?

Reply:
Agreed. I will remove the note from here and add a note to Ch2 of RM2.